Source/Private/ASElementMap.h

```diff
@@ -55,6 +60,11 @@ AS_SUBCLASSING_RESTRICTED
 - (nullable NSIndexPath *)indexPathForElement:(ASCollectionElement *)element;
 
 /**
+ * Returns the index path for the given element, if it represents an item. O(1)
+ */
+- (nullable NSIndexPath *)indexPathForElementIfItem:(ASCollectionElement *)element;
```

Comment:
I'd prefer -indexPathForElementIfRepresentsItem or the like. "IfItem" is kind of confusing. I couldn't understand its purpose until I read its documentation.